clusterloader2/pkg/tuningset/qps_load.go

```diff
@@ -14,32 +14,27 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package ticker
+package tuningset
 
 import (
 	"time"
 
+	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/perf-tests/clusterloader2/api"
 )
 
-type ticker struct {
-	C     <-chan struct{}
-	timer *time.Timer
-}
+type qpsLoad api.QpsLoad
```

Comment:
This is technically correct, but I'm not a fan of such construct.
Instead, can you do simply:

type qpsLoad struct {
   config *api.QpsLoad   (maybe config is not the best name)
}

Reply:
Done.